docs/admin_docs/adm_docs.md

```diff
@@ -0,0 +1,62 @@
+## Cel i zakres
+Poniższy rozdział ma na celu przedstawienie informacji wymaganych do efektywnego administrowania *Systemem Rezerwacji Biletów*. Tekst obejmuje zagadnienia dotyczące zarządzania od strony technicznej oraz obsługę dokumentacji systemu. 
+
+## Opis systemu
+
+### Software systemu
+| __Software__              | Opis                       | Technologie
+| :----------------------| :--------------------------| :--------------------------  
+| __Web UI__             | Graficzny interfejs użytkownika w postaci aplikacji webowej, front-end.     | React, TypeScript
+| __Serwer API__         | Serwer back-end serwujący dane systemowe poprzez API.                       | Java, Spring Boot  
+| __Baza danych__        | Instancja bazy danych, używana w celu osiągnięcia persystencji danych.      | PostgreSQL  
+| __Narzędzie konteneryzacji__   | Służy do automatyzacji wdrożenia oraz uruchomienia systemu          | Docker
+| __Generator dokumentacji__     | Narzędzie służące do budowy dokumentacji w postaci statycznych stron internetowych   | MkDocs 
+
+### Hardware systemu
+System docelowo instalowany oraz uruchamiany jest w ramach jednej fizycznej maszyny, na której konteneryzowane są poszczególne moduły systemu.
+
+## Komunikacja i dostępność
+Aplikacja kliencka domyślnie dostępna jest pod portem `:3000` oraz osiągalna przy pomocy przeglądarki internetowej z dowolnej maszyny z dostępem do internetu. Komunikacja z serwerem API (domyślnie port `:8080`) odbywa się protokołem HTTP. Serwer bazy danych znajduje się domyślnie pod portem `:5432`.
+
+## Instalacja systemu
+Istalacja systemu sprowadza się do umieszczenia plików źródłowych systemu w docelowym katalogu na maszynie fizycznej.
+
+W celu uruchomienia systemu należy zainstalować odpowiednie oprogramowanie. Wymagana jest instalacja narzędza [*Docker*](https://www.docker.com/) w wersji na system operacyjny maszyny fizycznej. Dokumentacja serwowana jest użytkownikom przy pomocy narzędzi [*MkDocs*](https://pypi.org/project/mkdocs/), [*MkDocs Material*](https://pypi.org/project/mkdocs-material/) oraz [*MkDocs with PDF*](https://pypi.org/project/mkdocs-with-pdf/), które muszą zostać zainstalowane na docelowej maszynie.
```

Comment:
"oraz [*MkDocs with PDF*](https://pypi.org/project/mkdocs-with-pdf/), które muszą zostać zainstalowane na docelowej maszynie." - a to jest prawda? MkDocs with PDF nie jest tylko do generowania pdf? jak ktoś nigdy pdf nie chce to chyba nie musi instalować tego drugiego plugina. Jeśli się nie mylę co do tego to proponuję tak to ubrać:

Dokumentacja serwowana jest użytkownikom przy pomocy narzędzia [*MkDocs*](https://pypi.org/project/mkdocs/), [*MkDocs Material*](https://pypi.org/project/mkdocs-material/). Istnieje też możliwość eksportowania dokumentacji do pliku PDF za pomocą [*MkDocs with PDF*](https://pypi.org/project/mkdocs-with-pdf/). Oba narzędzia przed użyciem muszą zostać zainstalowane na docelowej maszynie.